focus-areas/security/openssf-best-practices.md

```diff
@@ -1,41 +1,67 @@
 
-## Objectives
+The **OpenSSF Best Practices Badge**:
+- Indicates a project's compliance with "open source project best practices" as defined by the Linux Foundation’s core infrastructure initiative.
+- Encourages open source projects to produce secure software by adhering to established best practices.
+- Enables consumers to assess which open source projects are more likely to produce secure, reliable software based on their compliance with these best practices.
 
-As from the [OpenSSF Best Practices Badging Page](https://bestpractices.coreinfrastructure.org/en): OpenSSF badging indicates a project’s level of compliance with “open source project best practices" as defined by the Linux Foundation's core infrastructure initiative, which focuses on CyberSecurity in open source software. The goal of the program is to encourage projects to produce better more secure software and allow others to determine if a project is following best practices.
+## Want to Know More?
 
-Consumers of the badge can quickly assess which open source projects are following best practices and as a result are more likely to produce higher-quality secure software.
+<span markdown="1"><details>
+<summary>Click to read more about this metric.</summary>
 
-## Implementation
-*The usage and dissemination of health metrics may lead to privacy violations. Organizations may be exposed to risks. These risks may flow from compliance with the GDPR in the EU, with state law in the US, or with other law. There may also be contractual risks flowing from terms of service for data providers such as GitHub and GitLab. The usage of metrics must be examined for risk and potential data ethics problems. Please see [CHAOSS Data Ethics document](https://github.com/chaoss/community/blob/main/data-use-statement.md) for additional guidance.*
-
-See [OpenSSF’s API documentation](https://github.com/coreinfrastructure/best-practices-badge/blob/master/doc/api.md) for relevant information.
+### Data Collection Strategies
+As gotten from the [OpenSSF Best Practices Badging Page](https://bestpractices.coreinfrastructure.org/en), a project’s OpenSSF Best Practices status can be assessed when:
+- Projects receive a passing badge if they meet all required criteria.
+- The status of each criterion, for a given project, can be 'Met', 'Unmet', 'N/A' or 'unknown'.
+- Each criterion is in one of four categories: 'MUST', 'SHOULD', 'SUGGESTED', or 'FUTURE'.
+- To obtain a badge, all the MUST and MUST NOT criteria must be met, all SHOULD criteria must be met OR the rationale for not implementing the criterion must be documented, and all SUGGESTED criteria have to be rated as met or unmet.
+- Advanced badge levels of silver and gold are available if the project satisfies the additional criterion.
+
+For further information, refer to [OpenSSF’s API documentation](https://bestpractices.coreinfrastructure.org/en) 

```

Comment:
Can you change this URL to https://www.bestpractices.dev/en? Thank you!

Reply:
Done, thank you